README.rst

```diff
@@ -47,6 +47,11 @@ Make sure you have:
 * a valid JSON file, which contains the XPath expressions you want to check against the result tree
 * (optional: valid namespaces in the JSON and XML file)
 
+Options
+-------
+
+* ``--datadir``: directory which contains the XML and JSON files
+
```

Comment:
There is one information missing: you can add this option also in a configuration file. This can be `setup.cfg` or `pytest.ini`.